Mesos to KUDO blog post (#57)
kudobuilder/www: content/blog/blog-2019-10-02-kensipe.md

---
date: 2019-10-02
---

# From Mesos to KUDO

> When I started on KUDO, the first thing I need to wrap my head around was the difference between [Mesos](http://mesos.apache.org/) and [Kubernetes](https://kubernetes.io). I already had operator level experience with both, but my level of technical depth for Mesos was high and I needed a more intimate understanding of Kubernetes. In the process of coming up to speed I threw together guide for other Mesosnauts that I've been encourage could be useful for a wider audience. The details assume that you know or understand Mesos and it's eco-system and that you are new to Kubernetes.

Reviewer: ```suggestion
When I started on KUDO, the first thing I needed to wrap my head around was the difference between [Mesos](http://mesos.apache.org/) and [Kubernetes](https://kubernetes.io). I already had operator level experience with both, but my level of technical depth for Mesos was high and I needed a more intimate understanding of Kubernetes. In the process of coming up to speed I threw together a guide for other Mesonauts that could be useful for a wider audience. The details assume that you know or understand Mesos and its eco-system and that you are new to Kubernetes.
```
